FRAuth/FRAuth/OAuth2/AccessToken.swift

```diff
@@ -20,7 +20,7 @@ import Foundation
     /// token_type of access_token
     @objc public var tokenType: String
     /// Granted scope(s) with space separator for given access_token
-    @objc public var scope: String
+    @objc public var scope: String?
```

Comment:
Maybe leave that as non-optional and remove the scope from the validation below. So the flow actually passes without breaking the public API. If you want what you can do is set this with an empty string before doing all the checks. Like:
let scope = tokenResponse[OAuth2.scope] as? String ?? ""

Reply:
@george-bafaloukas-forgerock made the change. Please have a look again